diagrams-demo-gallery/demos/demo-custom-action/index.tsx

```diff
@@ -4,23 +4,16 @@ import createEngine, { DiagramModel, DefaultNodeModel, DefaultLinkModel } from '
 import { CanvasWidget, Action, ActionEvent, InputType } from '@projectstorm/react-canvas-core';
 import { DemoCanvasWidget } from '../helpers/DemoCanvasWidget';
 
-interface CustomDeleteItemsActionOptions {
-	keyCodes?: number[];
```

Comment:
I think we should still allow the user to specify the name of the keys. For example, it would be nice if they could specify `Del`, `Backspace` etc..

Reply:
Looks like you are right =)
But this feature will be disabled soon (not so soon, but still..)

I just though that codebase should be consistent with latest tendentions of the goodness of the code accepted in the community =)

Thanks for feedback!